src/server.ts

```
@@ -0,0 +1,886 @@
+              content: [
+                {
+                  type: 'text',
+                  text: this.safeStringify(response),
+                },
+              ],
+            };
+          }
+            
+          default:
+            // Delegate to individual tools
+            const toolResult = await this.orchestrator.executeTool(name, args);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)